app/models/bridge.rb

```diff
@@ -51,12 +51,12 @@ class Bridge < ApplicationRecord
   attr_readonly :slug
 
   def add_event_info
-    id, completed_at = latest_event_data
-
+    id, completed_at, created_at = latest_event_data
     attributes.merge!({
                         'eventCount' => events.count,
                         'completedAt' => completed_at || '',
-                        'eventId' => id
+                        'eventId' => id,
+                        'latestRequest' => created_at
```

Comment:
This is suppose to be for the latest inbound payload drop down right? If so, this should be `data['inbound']` not created_at.

Reply:
Also, mind adding a spec to ensure the controller returns the proper data? I think we are only testing if the controller works and not if the data returned is what is expected.